src/host/ut_host/ScreenBufferTests.cpp

```diff
@@ -907,7 +907,7 @@ void ScreenBufferTests::VtResize()
         L" The viewport should be w,h=80,30"));
 
     std::wstring sequence = L"\x1b[8;30;80t";
-    stateMachine.ProcessString(&sequence[0], sequence.length());
+    stateMachine.ProcessString(sequence);

```

Comment:
Is there any reason why we don't just pass the literal string sequence directly to the `ProcessString`, rather than assigning it to a variable first? I could understand if the sequence was being reused multiple times, but I think most of these are just used once and never again.

Reply:
It should go straight in. You are correct. I'll fix that up.

Reply:
I did a lot of these. Probably not all of them, but enough to make a statement.